docs/graphcast/radios/subgraph-radio.md

```diff
@@ -290,33 +286,33 @@ The Radio fetches new active allocations at a regular interval to ensure that it
 sequenceDiagram
     participant Network Subgraph
     participant Graph Node
-    participant POI Radio
+    participant Subgraph Radio
     participant Graphcast Network
     actor Human
     loop Track allocated deployments
-        POI Radio->>+Network Subgraph: Get latest allocated deployments
-        Network Subgraph->>-POI Radio: Return allocated deployments
+        Subgraph Radio->>+Network Subgraph: Get latest allocated deployments
```

Comment:
absolutely agreed, though I think we should leave it for when the Subgraph Versioning functionality is done so that we can add the web client piece to the diagram

Reply:
The bulk of this could be considered done in my opinion. The leftover pieces are preparation of the message from the web UI, and post-processing to indexer agent. I agree the two ends are essential for the users, but I think the pieces can be added in later whenever they are ready. In general I just think it would be good to keep the diagram updated accurately

Reply:
The diagram looks a bit too crowded now:( 
imo you won't need to add two more entity (`one-shot` and `subgraph developer` specifically), but simply `versionUpgradeMessage` emit from GraphcastNetwork to the Subgraph radio and have it be handled by notifying the human operator

Reply:
later on we can have a separate diagram that goes through the interaction between subgraph developer's messages and indexer's subgraph radio, with the specifics of developers using either one-shot radio or graphcast-web UI